.gitignore

```diff
@@ -11,3 +11,5 @@ test/integration/targets/web3-v1/types/web3-contracts
 test/integration/targets/ethers/types/ethers-contracts
 test/package-test/types
 packages/core/README.md
+.vscode
```

Comment:
use your global gitignore file please :) otherwise this file will become global list of all IDE's cache directories